README.md

```diff
@@ -146,9 +146,13 @@ On Windows the config file is located at:
 ## Issues (known, unknown, feature requests, etc.)
 
 If you run into a problem with Alacritty, please file an issue. If you've got a
-feature request, feel free to ask about it. Keep in mind that Alacritty is very
-much not looking to be a feature-rich terminal emulator with all sorts of bells
-and widgets.
+feature request, feel free to ask about it. Please just keep in mind Alacritty
+is focused on simplicity and performance, and not all features are in line with
+that goal.
+
+Before posting an issue try to give the existing issues a read over, there's a
+chance someone else has already reported it, and you can subscribe to that
+issue to keep up on the latest developments.
```

Comment:
`try to give the existing issues a read over` sounds a bit awkward I think.

I've got another proposal, though I'm not sure if it's not too 'forceful'. I'd rather have a duplicate than an issue that hasn't been reported.

```
Before opening a new issue, please make sure if it has already been reported. You can subscribe to existing issues to stay up to date with their progress.
```

Reply:
`please make sure if it has already been reported` sounds a little awkward as well.

I think this flows better, IMHO:
`Before opening a new issue, check if it has already been reported. <...>`

Reply:
Throw another `please` in front of the `check`  and it sounds awesome to me!

Reply:
Yep, that was my plan.